.circleci/config.yml

```diff
@@ -1842,14 +1858,33 @@ workflows:
           context:
             - slack
 
-  scheduled-fpp:
+  develop-fault-proofs:
     when:
-      equal: [ build_four_hours, <<pipeline.schedule.name>> ]
+      equal: [ "develop", <<pipeline.git.branch>> ]
     jobs:
+      - go-mod-download
+      - cannon-prestate:
+          requires:
+            - go-mod-download
+      - pnpm-monorepo:
+          name: pnpm-monorepo
+          requires:
+            - go-mod-download
       - fpp-verify:
           context:
             - slack
             - oplabs-fpp-nodes
```

Comment:
Sorry to be picky - I suspect `fpp-verify` needs to stay on a schedule as it takes nearly an hour to run.  But would be good to have the cannon tests run on every merge. We could run fpp-verify on every build but I suspect it will drive up costs a lot for not much gain in value.

Reply:
Ah ok! If it takes an hour then I'll definitely move it back.